Documentation/configuring-hive-metastore.md

```diff
@@ -35,36 +35,43 @@ Use [metastore-storage.yaml][metastore-storage-config] as a template and adjust
 By default to make installation easier Metering configures Hive to use an embedded Java database called [Derby](https://db.apache.org/derby/#What+is+Apache+Derby%3F), however this is unsuited for larger environments or metering installations with a lot of reports and metrics being collected.
 Currently two alternative options are available, MySQL and PostgreSQL, both of which have been tested with the metering-operator.
 
-There are 4 configuration options you can use to control the database used by Hive metastore: `url` , `driver` , `username` , and `password`.
+There are 4 configuration options you can use to control the database used by Hive metastore: `url` , `driver` , and `secretName` .
+The `url` is the clusterIP of the service, port followed by the database name.
+The `driver` is the database driver.
+The `secretName` is the name assigned to the secret which contains the username and password for the database.
 
-Using MySQL:
+## Using MySQL
```

Comment:
```suggestion
### Using MySQL
```

Reply:
This is also inconsistent with the existing "Use PostgreSQL for the Hive Metastore database" header that follows after these examples.